roles/hosted_engine_setup/README.md

```diff
@@ -47,6 +47,7 @@ Ansible role for deploying oVirt Hosted-Engine
 | he_debug_mode | false | If `True`, HE deployment will execute additional tasks for debug |
 | he_db_password | UNDEF | Engine database password |
 | he_dwh_db_password | UNDEF | DWH database password |
+| he_enable_grafana | False | If `True`, Grafana integration will be set up. Valid for oVirt >= 4.4. |
```

Comment:
Maybe add `on the engine VM` for clarification:
...Grafana integration will be set up on the engine VM...